packages/teams-js/src/public/meeting.ts

```diff
@@ -442,12 +461,29 @@ export namespace meeting {
    * @param handler The handler to invoke when the selfParticipant's raiseHand state changes.
    */
   export function registerRaiseHandStateChangedHandler(
-    handler: (raiseHandStateChangedEvent: IRaiseHandStateChangedEventData) => void,
+    handler: (eventData: RaiseHandStateChangedEventData) => void,
   ): void {
     if (!handler) {
       throw new Error('[registerRaiseHandStateChangedHandler] Handler cannot be null');
     }
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.raiseHandStateChanged', handler);
   }
+
+  /**
+   * Registers a handler for receiving meetingReactions. If any participant sends a meetingReaction, the participantId
+   * and meetingReactionTypw will be populated. Only one handler can be registered
```

Comment:
I don't see `participantId` be part of `MeetingReactionReceivedEventData`, so either the comment is incorrect, or you're missing some code.

Reply:
Also, there's a typo in meetingReacitonTyp**_w_** here

Reply:
Updated the comment